src/middlewared/middlewared/plugins/jail.py

```diff
@@ -284,6 +284,17 @@ def exec(self, jail, command, options):
 
         return msg.decode("utf-8")
 
+    @accepts(Dict("options", Str("jail"), Str("name")))
```

Comment:
How about we make this two parameters as well?

Its nice to have the identifiers in CRUDService always be the first parameter, that way the API is uniform and calls like this will work:

/api/v2.0/jail/id/jail_name/rename